website/content/docs/job-specification/resources.mdx

```diff
@@ -34,12 +34,15 @@ job "docs" {
 
 - `cpu` `(int: 100)` - Specifies the CPU required to run this task in MHz.
 
-- `cores` <code>(`int`: &lt;optional&gt;)</code> <sup>1.1 Beta</sup> - Specifies the number of CPU cores to reserve
-  for the task. This may not be used with `cpu`.
+- `cores` <code>(`int`: &lt;optional&gt;)</code> - Specifies the number of CPU cores
+  to reserve specifically for the task. This may not be used with `cpu`. The behavior
+  of setting `cores` is specific to each task driver (e.g. [docker][docker_cpu], [exec][exec_cpu]).
```

Comment:
I wonder if it would make sense to expand task driver [capabilities](https://nomad-f7bhb8tj9-hashicorp.vercel.app/nomad/docs/concepts/plugins/task-drivers#capabilities-capabilities-error) to include core reservation. That way we would be able to validate if the a task uses `cores` with a task driver that doesn't support it.

Reply:
Interesting idea - added to https://github.com/hashicorp/team-nomad/issues/373 along with thinking of PSI uses